sbt-akka-serialization-helper/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperPlugin.scala

```diff
@@ -19,6 +19,9 @@ object AkkaSerializationHelperPlugin extends AutoPlugin {
 
   override def globalSettings: Seq[Def.Setting[_]] = Nil
 
+  override def buildSettings: Seq[Def.Setting[_]] =
+    Seq(ashCompilerPluginCacheDirectory := baseDirectory.value / "target")
```

Comment:
This resolves to `ThisBuild / ashCompilerPluginCacheDirectory := ThisBuild / baseDirectory.value / "target"`

Reply:
Okay... so AFAICS now both `codec-registration-checker-cache.csv` and `dump-persistence-schema-cache/` are only located in top-level `target/`, and **not** in `target/` of sbt modules where ash is enabled...

Reply:
Would it be possible to address https://github.com/VirtusLab/akka-serialization-helper/issues/130 by also using top-level `target/` as the location for `akka-persistence-schema.yaml`? Coz having the same yaml duplicated across N different locations (for each module that enables ash) is a bit awkward&confusing